docs/users-manual/job-sets.rst

```diff
@@ -36,29 +36,34 @@ For example, if your jobs are described in a file named *my-jobs.set*:
 .. code-block:: condor-submit
    :caption: my-jobs.set file
 
-    name = MyJobSet
+   # Jobset example from https://htcondor.readthedocs.io/en/latest/users-manual/job-sets.html
+   name = MyJobSet
 
-    iterator = table inputfile {
-        input_A.txt
-        input_B.txt
-    }
-          
-    job {
-        executable = a.out
-        arguments = $(inputfile)
-        transfer_input_files = $(inputfile)
-        Request_cpus = 1
-        Request_memory = 1024M
-        Request_disk   = 1024M
-    }
+   iterator = table my_var {
+       input_A.txt
+       input_B.txt
+   }
+   
+   job {
+       executable     = /bin/echo
+       arguments = $(my_var)
```

Comment:
Since we aren't doing transfer input on the table iteration, should we change the item data to not be filenames?

Reply:
I think we can live with what we've got here.